Source/JavaScriptCore/dfg/DFGFixupPhase.cpp

```diff
@@ -2718,14 +2718,17 @@ class FixupPhase : public Phase {
             break;
         }
 
+        case NumberToStringWithValidRadixConstant:
         case NumberToStringWithRadix: {
             if (node->child1()->shouldSpeculateInt32())
                 fixEdge<Int32Use>(node->child1());
             else if (node->child1()->shouldSpeculateInt52())
                 fixEdge<Int52RepUse>(node->child1());
             else
                 fixEdge<DoubleRepUse>(node->child1());
```

Comment:
I think we should add `ToString` conversion code in this fixup phase if we successfully speculate on these types & it is `NumberToStringWithValidRadixConstant` and if the radix is 10.